src/reactors/V2DutchOrderReactor.sol

```diff
@@ -107,8 +107,8 @@ contract V2DutchOrderReactor is BaseReactor {
     }
 
     /// @notice validate the dutch order fields
-    /// - deadline must be greater than or equal than decayEndTime
-    /// - decayEndTime must be greater than or equal to decayStartTime
+    /// - deadline must be greater than or equal to decayEndTime
+    /// - decayEndTime must be greater than decayStartTime
```

Comment:
so orders with no decay should have same start/end amounts but must still specify different decay start and end times?

Reply:
yeah, it doesn't really matter what they are if start/end amounts are the same though